### PR TITLE
Fix pack: Code name WHITESPACE

### PR DESCRIPTION
**Pull request by IepIweidieng**
Adjust the white spaces by tool with manual reviews, fixed typos, and did more minor fixes.
Multi-statement macros are rewritten as well.

The punishment log message for violation of the bbs rule is also improved.

The neckers are fine-tuned as well.

### CODE REVIEW

**Comment by holishing** [comment]
There seem to be some defects to let ANSI color broken when using `pmore`.

**Comment by holishing** [comment]
btw, `footer` message in `maple/menu.c`, is shown by `outf()` function in master branch instead for testing wide screen

File: maple/menu.c [hunk 1]
```diff
@@ -414,67 +414,67 @@ static char footer[160];
...
-  /* Thor.980913: ���: �̱`���I�s movie()���ɾ�O�C����s film, �b 60��H�W,
-                        �G���ݰw�� xx:yy �ӯS�O�@�@�r���x�s�H�[�t */
+    /* Thor.980913: ���: �̱`���I�s movie()���ɾ�O�C����s film, �b 60��H�W,
+                          �G���ݰw�� xx:yy �ӯS�O�@�@�r���x�s�H�[�t */
 
-  sprintf(footer, "\033[0;34;46m%s%d:%02d] \033[30;47m �ثe���W��\033[31m%4d\033[30m �H, �ڬO \033[31m%-12s\033[30m [�I�s/�T��]\033[31m%s",
-    datemsg, ufo / 60, ufo % 60,
-    /*ushm->count*/total_num, cuser.userid, flagmsg);
-  outf(footer);
+    sprintf(footer, "\033[0;34;46m%s%d:%02d] \033[30;47m �ثe���W��\033[31m%4d\033[30m �H�A�ڬO \033[31m%-12s\033[30m [�I�s/�T��]\033[31m%s\033[m",
```
Comment (holishing) [inline maple/menu.c]: The reason I removed `\033[m` in 229a2d2 is that `outf()` function will automatically provides `\033[m` in the end of output message.
maybe we can add notes at that part later, or find better way to rewrite it.

File: maple/pmore.c [hunk 2]
```diff
@@ -193,23 +193,23 @@
 
 // Colors
 #define PMORE_COLOR_HEADER1 \
-    ANSI_COLOR(47;34)
+    ANSI_COLOR(47; 34)
 #define PMORE_COLOR_HEADER2 \
```
Comment (holishing) [inline maple/pmore.c]: I think this part should be recovered because of those defects discussed in comment ( #26 ).

Comment (IepIweidieng) [reply maple/pmore.c]: Resolved in d95ff15.

File: maple/menu.c (hunk 1 above)
Comment (IepIweidieng) [reply maple/menu.c]: The `\033[m` was removed again in 229a2d2.

**Comment by holishing** [comment]
Ok, I'll merge later, thanks for your contribution!!

**Pull request merged**